sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -156,15 +185,58 @@ def test_mini_batch_fit_transform():
         joblib_par.multiprocessing = None
         try:
             U2 = MiniBatchSparsePCA(n_components=3, n_jobs=2, alpha=alpha,
-                                    random_state=0).fit(Y).transform(Y)
+                                    random_state=0,
+                                    normalize_components=norm_comp)\
+                .fit(Y).transform(Y)
```

Comment:
avoid that `\`. Just create an instance and then call fit_transform on a second line. It is more readable.